### ISSUE TITLE
New attack technique: Use GCP OSConfig (VM Manager) to Execute Code

### ISSUE THREAD

**Issue by siigil** [DataDog/stratus-red-team#534]
Use VM Manager to Execute Code: GCP
https://unit42.paloaltonetworks.com/cloud-virtual-machine-attack-vectors/

Suggested tags: kind/new-technique, platform/gcp

**Comment by rileydakota** [comment]
@christophetd thoughts on this? Would it be worth modelling it after one of these? 
https://stratus-red-team.cloud/attack-techniques/AWS/aws.discovery.ec2-enumerate-from-instance/ 
https://stratus-red-team.cloud/attack-techniques/AWS/aws.execution.ssm-start-session/

**Comment by christophetd** [comment]
Absolutely, I think it makes sense to implement it like aws.execution.ssm-start-session
